test/functionalTest/cases/0864_filter_string_scope/filter_string_scope.test

```diff
@@ -175,18 +175,44 @@ Date: REGEX(.*)
 
 
 
-06. v1 Query: temperature<60;humidity==83..61;status==ERR -> E2
+06. v1 Query: temperature<60;humidity==61..83;status==ERR -> E2
 ===============================================================
 HTTP/1.1 200 OK
-Content-Length: 94
+Content-Length: 602
 Content-Type: application/json
 Date: REGEX(.*)
 
 {
-    "errorCode": {
-        "code": "404",
-        "reasonPhrase": "No context element found"
-    }
+    "contextResponses": [
+        {
+            "contextElement": {
```

Comment:
For attribute information that is stored as number of boolean in the database (due they have been introduces using v2 operations) it would be great if we could render as string in the v1 GET method, avoding the current `"value": ""`.

In this case, we would get something like this:

```
{
                "attributes": [
                    {
                        "name": "humidity",
                        "type": "",
                        "value": "70"
                    },
                    {
                        "name": "status",
                        "type": "",
                        "value": "ERR"
                    },
                    {
                        "name": "temperature",
                        "type": "",
                        "value": "30"
                    }
                ],
                "id": "E2",
                "isPattern": "false",
                "type": "T"
            }
```